documentation/developer-guide/modules/tooling-guide/pages/java-aspect-tooling.adoc

```diff
@@ -235,15 +405,45 @@ xref:bamm-specification:ROOT:payloads.adoc#mapping-to-json[mapping rules] as def
 The generator uses `bamm:exampleValue`&#8203;s of Properties if present, otherwise random values corresponding to the
 respective data types are generated.
 
-include::sds-developer-guide:ROOT:partial$sample-json-payload-generation.adoc[]
+++++
+<details>
+<summary>Show used imports</summary>    
+++++
+[source,java,indent=0,subs="+macros,+quotes"]
+----
+include::example$GenerateJsonPayload.java[tags=imports]
+----
+++++
+</details>
+++++
+
+[source,java,indent=0,subs="+macros,+quotes"]
+----
+include::example$GenerateJsonPayload.java[tags=generate]
+----
 
 === Generating JSON Schema
 
 The JSON schema generator creates a https://json-schema.org/[JSON Schema] that describes the payload for an Aspect model
 as it could be returned by an Aspect that implements that Aspect model. Currently, the generated schema corresponds to
```

Comment:
corresponds to ???? 
-> isn't something missing here? (the include has been removed)

Reply:
This was actually broken before as well:
![image](https://user-images.githubusercontent.com/1199036/215424828-64881c6d-b5f0-4fd6-b066-2d6a24bad8e9.png)